### PR TITLE
Update src/lib/seam/connect via 22b29be2bc9fa98746ca4be25b5a2e513d0a0126

### PR DESCRIPTION
**Pull request by seambot**

### CODE REVIEW

File: src/lib/seam/connect/route-types.ts [hunk 1]
```diff
@@ -535,7 +535,7 @@ export interface Routes {
       client_session: {
         token: string
         client_session_id: string
-        created_at: string
+        created_at: string | Date
```
Comment (razor-x) [inline src/lib/seam/connect/route-types.ts:538]: @codetheweb Is this right? All responses must be primitives right?

File: src/lib/seam/connect/route-types.ts [hunk 2]
```diff
@@ -1343,7 +1343,7 @@ export interface Routes {
     jsonResponse: {
       ok: boolean
       msg: 'I\u2019m one with the Force. The Force is with me.'
-      last_service_evaluation_at: string
+      last_service_evaluation_at?: string | undefined
```
Comment (razor-x) [inline src/lib/seam/connect/route-types.ts:1346]: @codetheweb Why is this nullable now?

File: src/lib/seam/connect/route-types.ts (hunk 1 above)
Comment (codetheweb) [reply src/lib/seam/connect/route-types.ts:538]: would be good if we could lint against this somehow in seam connect

File: src/lib/seam/connect/route-types.ts (hunk 2 above)
Comment (codetheweb) [reply src/lib/seam/connect/route-types.ts:1346]: for https://github.com/seamapi/seam-connect/pull/3648, I fixed the types enough for it to pass without changing the actual values (i.e. was was previously optional but typed incorrectly)

File: src/lib/seam/connect/route-types.ts (hunk 1 above)
Comment (codetheweb) [reply src/lib/seam/connect/route-types.ts:538]: https://github.com/seamapi/seam-connect/pull/3657

**Pull request closed without merging**